KEMField/Source/Interface/KGeoBag/src/KGElectromagnetConverter.cc

```diff
@@ -226,7 +231,7 @@ void KGElectromagnetConverter::VisitCylinderTubeSpace(KGCylinderTubeSpace* cylin
                                               GlobalToInternalVector(fCurrentZAxis));
         fElectromagnetContainer->push_back(coil);
 
-        if (fMagfield3File) {
+        if (fMagfield3File->IsOpen()) {
```

Comment:
I think this should also check for nullptr since it might not be initialized at this point: `if (fMagfield3File && fMagfield3File->IsOpen())`